README.md

```diff
@@ -107,30 +124,37 @@ The sample lists all of the storage accounts in a given subscription:
 
 ```csharp
 //Get all the storage accounts for a given subscription
-IEnumerable<StorageAccount> storAcctsSub = storageMgmtClient.StorageAccounts.List();
+AsyncPageable<StorageAccountResource> storAcctsSub = subscription.GetStorageAccountsAsync();
 ```
 
 It also lists storage accounts in the resource group:
 
 ```csharp
-//Get the storage account keys for a given account and resource group
-IList<StorageAccountKey> acctKeys = storageMgmtClient.StorageAccounts.ListKeys(rgName, accountName).Keys;
+//Get a list of storage accounts within a specific resource group
+AsyncPageable<StorageAccountResource> storAccts = resourceGroup.GetStorageAccounts().GetAllAsync();
 ```
 
-### Read and regenerate storage account keys
+### List or regenerate storage account keys
 
-The sample lists storage account keys for the newly created storage account and resource group:
+The sample lists storage account keys for the newly created storage account:
 
 ```csharp
 //Get the storage account keys for a given account and resource group
-IList<StorageAccountKey> acctKeys = storageMgmtClient.StorageAccounts.ListKeys(rgName, accountName).Keys;
+StorageAccountGetKeysResult result = storageAccount.GetKeys();
+IReadOnlyList<StorageAccountKey> acctKeys = result.Keys;
+Console.WriteLine($"List of storage account keys in {storageAccount.Id.Name}:");
+foreach (StorageAccountKey acctKey in acctKeys)
+{
+    Console.WriteLine($"\t{acctKey.KeyName}");

```

Comment:
Should we print only the key name but not key vault here? (normally, customer will care about key value more)
As the before code doesn't show how to print the keys, we might also don't need to print it?